.golangci.yml

```diff
@@ -1,47 +1,27 @@
+run:
+  deadline: 5m
+  skip-files:
+    # Skip autogenerated files.
+    - ^.*\.(pb|y)\.go$
+  skip-dirs:
+    - "vendor$"
+
+output:
+  sort-results: true
+
 linters:
-  disable-all: true
   enable:
-    - prealloc
-    - ineffassign
-    - goerr113
+    - depguard
+    - goimports
+    - revive
     - misspell
-    - errcheck
-    - gosimple
-    - staticcheck
-    - gosec
-    - gocritic
-    - unparam
-    - deadcode
```

Comment:
should we keep this? and the `exportloopref` as well 🤔

Reply:
Right. Let me revert them. (Just commented out them to check the action before.)